start_esp/nginx-auto.conf.template

```diff
@@ -48,10 +48,7 @@ http {
 % endif
 % endfor
 
-  set_real_ip_from  0.0.0.0/0;
-  set_real_ip_from  0::/0;
-  real_ip_header    X-Forwarded-For;
-  real_ip_recursive on;
```

Comment:
can we have both?  For GCP, we can use X-User-IP,  for others,  we may have to use XFF